client/src/components/WelcomeBadge.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Label, Flag } from 'semantic-ui-react';
+import { flagCodeByCountry } from '../flags';
+import FootyPoints from './FootyPoints';
+
+function WelcomeBadge(props) {
+    const { role, name, country, footyPoints } = props;
+    const readableRole = {
+        'ALUMNI': 'Alumnus',
+        'STUDENT': 'Student',
+        'COUNTRY_AMBASSADOR': 'Ambassador',
+        'ADMIN': 'Admin'
+    };
+    const showFlag = (role === 'COUNTRY_AMBASSADOR');
+
+    let welcomeMessage = '';
+    if (role === 'COUNTRY_AMBASSADOR') {
+        welcomeMessage = `Welcome ${readableRole[role]} ${name}`;
+    } 
+    else {
+        welcomeMessage = `Welcome ${name} (${readableRole[role]})`;
+    }
+
+    return (
+        role ? 
+            <div>
+                <Label style={{
+                    backgroundColor: 'black',
+                    color: 'white'
+                }}>
+                    {welcomeMessage}
+                    {showFlag ?
```

Comment:
The formatting here is off can we fix this?
![image](https://user-images.githubusercontent.com/31665569/109390953-9accff00-78e2-11eb-9701-cdb85b12e72e.png)

So it has flag first and appropriate padding like so
![image](https://user-images.githubusercontent.com/31665569/109390971-ac160b80-78e2-11eb-8a28-6ab044808f9d.png)